aeppl/transforms.py

```diff
@@ -422,8 +422,20 @@ def transform(measurable_input, *other_inputs):
 def measurable_reciprocal(fgraph, node):
     """Rewrite a `reciprocal` node to a `MeasurableVariable`."""
 
-    def transform(measurable_input, *other_inputs):
-        return ReciprocalTransform(), (measurable_input,)
+    new_node = at.power(node.inputs[0], at.as_tensor(-1)).owner
```

Comment:
What's the advantage of using `pow` to represent `reciprocal` here (besides a few less lines of code)? I feel such equivalences should be handled at the Aesara-level, either by aliasing or a canonicalization pass.